packages/redirections/migrations/sqls/20180103194546-redirections-up.sql

```diff
@@ -0,0 +1,11 @@
+create table "redirections" (
+  "id"                  uuid primary key not null default uuid_generate_v4(),
+  "source"              text not null,
+  "target"              text not null,
+  "status"              integer not null default 308,
```

Comment:
should be `301`

Reply:
only other regular value for redirect is `302`

Reply:
308, strange ;)